File: 5/5.c
#include <stdbool.h>
#include <stdio.h>

bool isdiv(long answer)
{
  for (int x=2; x < 21; x++) {
    if (answer % x) {
      return false;
    }
  }

  return true;
}

int main()
{
  long answer = 2520;

  while (!isdiv(answer)) {
    answer += 20;
  }

  printf("Problem 5 Answer: %ld\n", answer);
  return 0;
}

File: 5/5.go
package main

import "fmt"

func isdiv(i int) bool {
	for j := 2; j < 21; j++ {
		if i%j != 0 {
			return false
		}
	}
	return true
}

func main() {
	answer := 2520
	for !isdiv(answer) {
		answer += 20
	}
	fmt.Println("Problem 5 Answer:", answer)
}
